src/presets/next.js

```diff
@@ -47,6 +47,9 @@ export default {
             hover: "#333333",
             active: "#333333",
           },
+          sub: {
+            DEFAULT: "#f3f3f3",
```

Comment:
odd ot see capitalised properties when the rest is lower case

Reply:
Tailwind feature: https://tailwindcss.com/docs/theme#core-plugins